### PR TITLE
S2t3

### PR DESCRIPTION
**Pull request by EleanorRoberts**

### CODE REVIEW

File: config/routes.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
+
 const MarkerAPIController = require("../src/Controllers/MarkerAPIController");
+const AddNewRouteController = require("../src/Controllers/AddNewRouteController");
 
 function routes(app){
     app.get('/markers', MarkerAPIController.getAllStartMarkersController);
+    app.post('/', AddNewRouteController.addNewRouteController)
```
Comment (Roomsycakes01) [inline config/routes.js:7]: change the route!!

File: src/Controllers/AddNewRouteController/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const DbService = require("../../Services/DbService");
+
+async function addNewRouteController(request, response) {
+    const collection = await DbService.connectToDb();
+    const data = await DbService.addNewRoute(collection, request.body);
+}
+
+module.exports.addNewRouteController = addNewRouteController
```
Comment (Roomsycakes01) [inline src/Controllers/AddNewRouteController/index.js]: you know the drill

File: src/Controllers/AddNewRouteController/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const DbService = require("../../Services/DbService");
+
+async function addNewRouteController(request, response) {
+    const collection = await DbService.connectToDb();
+    const data = await DbService.addNewRoute(collection, request.body);
```
Comment (Roomsycakes01) [inline src/Controllers/AddNewRouteController/index.js]: do we wanna check if the body has stuff in it?

**Review by Roomsycakes01 — COMMENTED** [review]
lets go!!

File: src/Controllers/AddNewRouteController/index.js (hunk 3 above)
Comment (EleanorRoberts) [reply src/Controllers/AddNewRouteController/index.js]: That'll be dealt with in validation

**Pull request merged**